README.md

```diff
@@ -10,6 +10,8 @@ An Ansible Playbook Bundle (APB) is a new method for defining and distributing c
 
 Read more about the Ansible Service Broker and Ansible Playbook Bundles in this [introduction](docs/introduction.md).
 
+**NOTE**: this broker will be based on the [open-service-broker-sdk](https://github.com/openshift/open-service-broker-sdk) project in the future.
```

Comment:
@jmrodri added a suggestion below.  Paul wanted us to warn new contributors about both the rebase to broker sdk as well as that code may drastically change so if they are submitting a PR it might need to be reworked.


**NOTE**: this broker will be re-based on the [open-service-broker-sdk](https://github.com/openshift/open-service-broker-sdk) project in the future.  Please be aware any PRs submitted may need to be reworked, best to contact us at ansible-service-broker@redhat.com before beginning a PR.